src/algorithms/matching.js

```diff
@@ -13,8 +13,8 @@ import sortBy from 'lodash-es/sortBy';
  * yields more intutive results.
  *
  * @param {number[][]} matrix the input matrix
- * @param {string} algorithm the algorithm to run; `hungarian` or `greedy`
  * @param {object} parameters additional parameters to forward to the algorithm
+ * @param {string} parameters.algorithm the algorithm to run; `hungarian` or `greedy`
```

Comment:
This is weird; did the linter complain about the existing docstring? Strictly speaking, `algorithm` is not part of `parameters` because `parameters` itself is just the "rest" argument of the options object.

Reply:
The linter complained where I called this function. I guess since the second argument has no name, it automatically interpreted it as `parameters` from the docstring? No idea actually, but this change fixed the issue.